<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainW</class>
 <widget class="QWidget" name="MainW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainW</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>设定</string>
   </property>
   <widget class="QGroupBox" name="methods">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>功法</string>
    </property>
    <widget class="QListWidget" name="MartialMethodList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="medicine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>使用丹药</string>
    </property>
   </widget>
   <widget class="QPushButton" name="magicWeapons">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>法宝加持</string>
    </property>
   </widget>
   <widget class="QPushButton" name="list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>全球排行榜</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>境界：</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>256</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>未知</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>信息</string>
    </property>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>昵称：</string>
     </property>
    </widget>
    <widget class="QLabel" name="stateLabel_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>16</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>未知</string>
     </property>
    </widget>
    <widget class="QPushButton" name="register_">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
